app/src/main/java/com/willowtree/vocable/settings/EditCategoriesListFragment.kt

```diff
@@ -161,6 +161,11 @@ class EditCategoriesListFragment : BaseFragment() {
             moveCategoryDownButton.action = {
                 editCategoriesViewModel.moveCategoryDown(category)
             }
+
+            if (category?.categoryId == getString(R.string.category_my_sayings_id)) {
+                editCategorySelectButton.isEnabled = false
```

Comment:
Do we want to disable it or make it invisible?

Reply:
Steve says we should disable it 🤷

Reply:
i think we should make it invisible tho, but maybe we can talk about it